manifests/r/Rambox/RamboxCE/0.7.8/Rambox.RamboxCE.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Rambox.RamboxCE
+PackageVersion: 0.7.8
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/ramboxapp/community-edition/releases/download/0.7.8/Rambox-0.7.8-win.exe
+  InstallerSha256: 603A7BA124687F0A4071CF2FCD68DED2FFA96F6FE381E193382A06B4E8E31BDD
+  UpgradeBehavior: install
```

Needed for upgrade compatibility.

```suggestion
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/ramboxapp/community-edition/releases/download/0.7.8/Rambox-0.7.8-win.exe
  InstallerSha256: 603A7BA124687F0A4071CF2FCD68DED2FFA96F6FE381E193382A06B4E8E31BDD
  InstallerSwitches:
    Custom: /CURRENTUSER
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/ramboxapp/community-edition/releases/download/0.7.8/Rambox-0.7.8-win.exe
  InstallerSha256: 603A7BA124687F0A4071CF2FCD68DED2FFA96F6FE381E193382A06B4E8E31BDD
  InstallerSwitches:
    Custom: /ALLUSERS
  UpgradeBehavior: install
```